Demo/Application/Base/SceneDelegate.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+@objcMembers class SceneDelegate: UIResponder, UIWindowSceneDelegate {
```

Comment:
Same question here on if we need `@objcMembers`

Reply:
Not needed! Addressed in dc311d2bb8860d88472b04f886b40b4358ac1c75